### PR TITLE
[INLONG-4356][Manager] Support create postgres resource

### LINKED ISSUE #4356: [Feature][Manager] Support create postgres resource
### Description

Support create postgres resource

### Use case

Support create postgres resource

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
Support create postgres resource

Fixes #4356

### CODE REVIEW

**Review by healchow — CHANGES_REQUESTED** [review]

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/PostgresStreamSinkServiceTest.java [hunk 1]
```diff
@@ -99,4 +109,53 @@ public void testGetAndUpdate() {
         deletePostgresSink(postgresSinkId);
     }
 
+//    @Test
```
Comment (healchow) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/PostgresStreamSinkServiceTest.java:112]: Just comment out `@Test` and add the note: turned on when testing locally.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/postgres/PostgresJdbcUtils.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+            Statement stmt = conn.createStatement();
+            LOG.info("execute sql [{}] success for url: {}", sql, url);
+            return stmt.execute(sql);
+        }
+    }
+
+    /**
+     * Execute One Postgres Sql command
+     */
+    public static ResultSet executeSql(String sql, String url, String user, String password) throws Exception {
```
[the remark this answers is not in the document]
Comment (baomingyu) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/postgres/PostgresJdbcUtils.java:84]: Please look at the implementation logic, which requires different return value types and different execution methods

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/postgres/PostgresJdbcUtils.java [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+        }
+        Connection conn;
+        try {
+            Class.forName(POSTGRES_DRIVER_CLASS);
+            conn = DriverManager.getConnection(url, user, password);
+        } catch (Exception e) {
+            LOG.error("get postgres connection error, please check postgres jdbc url, username "
+                    + "or password", e);
+            throw new Exception(
+                    "get postgres connection error, please check jdbc url, username or password. "
```
Comment (gong) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/postgres/PostgresJdbcUtils.java:61]: Content is same. May be throw exception is enough.

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/postgres/PostgresJdbcUtils.java [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+     */
+    public static void createDb(String url, String user, String password, String dbName)
+            throws Exception {
+        String checkDbSql = PostgresSqlBuilder.getCheckDatabase(dbName);
+        ResultSet resultSet = executeSql(checkDbSql, url, user, password);
+        if (resultSet != null) {
+            resultSet.next();
+            if (resultSet.getInt("count") == 0) {
+                String createDbSql = PostgresSqlBuilder.buildCreateDbSql(dbName);
+                executeCheckSql(createDbSql, url, user, password);
```
Comment (gong) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/postgres/PostgresJdbcUtils.java:122]: function name have ambiguity in here to use.

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/PostgresStreamSinkServiceTest.java [hunk 5]
```diff
@@ -99,4 +107,53 @@ public void testGetAndUpdate() {
         deletePostgresSink(postgresSinkId);
     }
 
+    /**
+     *  just using in local test
+     */
+    //@Test
+    public void testDbResource() {
```
Comment (gong) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/core/sink/PostgresStreamSinkServiceTest.java:114]: use @Ignore("just using in local test")

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**